drivers/input/button_upper.c

```diff
@@ -138,7 +140,7 @@ static const struct file_operations btn_fops =
   btn_open,  /* open */
   btn_close, /* close */
   btn_read,  /* read */
-  NULL,      /* write */
+  btn_write, /* write */
```

Comment:
Same comment: normally an ordinary button shouldn't have write operation, it should depend on some config.